spec/user_spec.rb

```diff
@@ -6,5 +6,10 @@
     expect(user).not_to be nil
   end
 
-  # Write a new unit test here.
-end
\ No newline at end of file
+  it 'should create a name for the user' do
+    user = User.create
+    user.name = 'Amber'
+    user.save
+    expect(user.name).to eq 'Amber'
+  end
```

Comment:
This is partially my fault because I made the step above do nothing, but this expectation doesn't actually test anything. We are setting the `user.name` above, so of course it is `Amber`. A better test would be `User.last.name` is `Amber`, right? That would confirm we have saved the file in the database.